docs/src/content/docs/id/reference/frontmatter.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Referensi Frontmatter
+description: Ikhtisar bidang frontmatter default yang didukung Starlight.
+---
+
+Anda dapat menyesuaikan masing-masing halaman Markdown dan MDX di Starlight dengan menetapkan nilai di frontmatter. Sebagai contoh, halaman biasa dapat mengatur bidang `title` dan `description`:
+
+```md
+---
+title: Tentang proyek ini
+description: Pelajari lebih lanjut tentang proyek yang sedang Saya kerjakan.
+---
+
+Selamat datang di halaman Tentang!
+```
+
+## Bidang Frontmatter
+
+### `title` (diperlukan)
+
+**type:** `string`
+
+Anda harus memberikan judul untuk setiap halaman. Ini akan ditampilkan di bagian atas halaman, di tab browser, dan di metadata halaman.
+
+### `description`
+
+**type:** `string`
+
+Deskripsi halaman digunakan untuk metadata halaman dan akan diambil oleh mesin pencari dan dalam pratinjau media sosial.
+
+### `editUrl`
+
+**type:** `string | boolean`
+
+Mengesampingkan [konfigurasi `editLink` global](/id/reference/configuration/#editlink). Atur ke `false` untuk menonaktifkan tautan "Edit halaman" untuk halaman tertentu atau memberikan URL alternatif di mana konten halaman ini dapat diedit.
+
+### `head`
+
+**type:** [`HeadConfig[]`](/id/reference/configuration/#headconfig)
+
+Anda dapat menambahkan tag tambahan ke `<head>` halaman Anda menggunakan bidang `head` frontmatter. Ini berarti Anda dapat menambahkan gaya khusus, metadata, atau tag lain ke satu halaman. Mirip dengan [opsi `head` global](/id/reference/configuration/#head).
+
+```md
+---
+title: Tentang kami
+head:
+  # Gunakan tag <title> khusus
+  - tag: title
+    content: Halaman tentang khusus
+---
+```
+
+### `tableOfContents`
+
+**type:** `false | { minHeadingLevel?: number; maxHeadingLevel?: number; }`
+
+Mengesampingkan [konfigurasi `tableOfContents` global](/id/reference/configuration/#tableofcontents).
+Sesuaikan tingkat judul yang akan disertakan atau atur ke `false` untuk menyembunyikan daftar isi pada halaman ini.
+
+```md
+---
+title: Halaman dengan hanya H2 dalam daftar isi
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
+---
+```
+
+```md
+---
+title: Halaman tanpa daftar isi
+tableOfContents: false
+---
+```
+
+### `template`
+
+**type:** `'doc' | 'splash'`  
+**default:** `'doc'`
```

Comment:
Same question like "type" above to be sure, is this fine to have "default" here? I know for example in French we change it but it may be totally fine in Indonesian.

Reply:
Actually, this can be written as "type" and "default". But I will try to translate it for better quality 👍